modules/extensions/src/test/java/org/apache/synapse/mediators/bsf/ScriptMediatorTest.java

```diff
@@ -63,84 +75,95 @@ public void testThreadSafety() throws Exception {
                 Integer.parseInt(randomno) * 2);
     }
 
+    public void testThreadSafetyOnNashornEngine() throws Exception {
```

Comment:
Add comment explaining the test.

Reply:
Hi,
Fixed with https://github.com/wso2/wso2-synapse/pull/971